tasks/need_fixing_linters/README.md

```diff
@@ -0,0 +1,39 @@
+
+# [HOWTO] Fix the linter errors for your team
+
+## Context
+
+Right after `7.48` merge freeze [we enabled `revive` and `gosimple` linters](). To avoid breaking `main` they are only linting commits made after [f40667d](https://github.com/DataDog/datadog-agent/commit/f40667d3841c6339be0d00d53e54a4a63f43f11e).
+
+We still want to fix the `revive` and `gosimple` errors that came before [f40667d](https://github.com/DataDog/datadog-agent/commit/f40667d3841c6339be0d00d53e54a4a63f43f11e).
+
+
+## Prerequisites
+
+The version of `go` and `golangci-lint` can have a big impact on the output of the linters. You'll need to have a setup close to the CI's to have the same output. Please meet the following conditions before running the commands:
+- Your `go` version (run `go version`) is identical to the CI's (content of the `.go-version` file).
+- Your `golangci-lint` version is identical to the CI's (look for `golangci-lint` in the `go.mod file`).
+- You did not install `go` using `brew` (`which go` path shouldn't contain `homebrew`).
+- Install the requirements with `python3 -m pip tasks/fix_revive_linter/install requirements-need-fixing-linter.txt
```

Comment:
```suggestion
- Install the requirements with `python3 -m pip tasks/fix_revive_linter/install requirements-need-fixing-linter.txt`
```

Reply:
Thanks ! Fixed in [6d20653](https://github.com/DataDog/datadog-agent/pull/19015/commits/6d2065315f7ba06acbcea87bde309fd014ca4c61).